.github/workflows/linux.yml

```diff
@@ -110,44 +110,30 @@ jobs:
       - name: Install Meson
         run: |
           sudo apt install -y -V meson ninja-build
-      - name: Configure PGroonga by Meson
+      - name: Configure PGroonga
         run: |
           meson setup ../pgroonga.build -Dpg_config=${PG_CONFIG}
         env:
           PG_CONFIG: "/usr/lib/postgresql/${{ matrix.postgresql-version }}/bin/pg_config"
-      - name: Build PGroonga by Meson
+      - name: Build PGroonga
         # TODO: Remove this when we're PostgreSQL 18 ready
         continue-on-error: ${{ matrix.postgresql-unreleased == 'yes' && true || false }}
```

Comment:
Can we remove these conditions?

Reply:
fix: [ffea45b](https://github.com/pgroonga/pgroonga/pull/753/commits/ffea45b3a1cd889c07c108d03b6b0162ccc6024b)